### ISSUE TITLE
Color of images during labelling

### ISSUE THREAD

**Issue by harshk95** [diegoaldarondo/Label3D#15]
Hi, 
I am using Label3D to label multiple camera views for hand labeling for use in DANNCE.
I have mono videos recorded using Basler cameras and when I open the frames in Label3D they appear in a strange yellow/blue color instead of the the expected greyscale. 
Strangely when I pick the first 100 frames to labeled as oppposed to 100 random frames, the first 2 views are in the expected color scale while others are not. With the random frames, all the views are in the blue-yellow hue. 
It would be helpful for ease of use, if this could be controlled and I could view the frames in the original color scale while labeling them.

Thanks.

**Comment by diegoaldarondo** [comment]
Good idea, I'll add an optional mono colormap option that defaults to grayscale.

**Comment by diegoaldarondo** [comment]
Added a branch with this feature. The optional argument is `monoColormap` and the default is `'gray'`. Let me know if that meets your needs and I'll work on merging into master. 

https://github.com/diegoaldarondo/Label3D/tree/monoColormap

**Comment by harshk95** [comment]
Hi,
It works well, just needed a small tweak, in the Label3D.m in:
https://github.com/diegoaldarondo/Label3D/blob/02fbe48cbfa1efc81254150e23b7862dac62aab4/Label3D.m#L295

len needed to be changed to length

Thanks  lot!

**Comment by diegoaldarondo** [comment]
Ha, sorry about that. Will fix and merge later.
